src/components/subscribemodal/index.tsx

```diff
@@ -113,13 +138,23 @@ const CustomSpan = styled.span`
   position: initial;
   inset: auto;
   z-index: auto;
+  margin-right: 10px;
+
+  @media (max-width: 600px) {
```

Comment:
we can create a constant called `BREAKPOINT_SMALL` maybe and use that everywhere wherever we are using media queries. we will revisit the media queries value all together at a later point.